projects/yfdai/index.js

```diff
@@ -0,0 +1,42 @@
+const Web3 = require('web3');
+require("dotenv").config();
+let web3RpcUrl = process.env.ETHEREUM_RPC;
+const web3 = new Web3(new Web3.providers.HttpProvider(web3RpcUrl));
+module.exports = web3;
+const retry = require('../helper/retry')
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+const abi = require('./abi');
+
+async function fetch() {
+
+    const yfdaiTokenAddress = "0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577";
+
+    const YfDaiETHLP72HRSAddress = "0x75E9F410e8d1D7240b67ec6FE35FA37580b814d9";
+    const YfDaiETHLP30DayAddress = "0x8D704D4107CBE5ebE8c0236C5506b30Bf8Bad305";
+    const YfDaiETHLP60DayAddress = "0x26572bf2620108cb5006987e6348c07dc4e14a0f";
+    const YfDaiETHLP90DayAddress = "0x175d6cbaeff93734ada4c5430815f2208a6b040c";
```

Comment:
The yfdai on these contracts seem to be rewards for the LPs staked within, is that correct?

Reply:
Yes, that's correct. 
All these are YFDAI Farms, Where users can stake LPs and get YFDAI in rewards.

Reply:
For other protocols we don't count the money used as rewards since those tokens are not even part of the circulating supply and have not been locked on a contract by anyone. To keep comparisons fair I believe it would be best not to include it here either.